rockstar/rockstar.js

```diff
@@ -970,6 +1000,210 @@
         }
     }
 
+    /////////////////////////////////////////////////////
+    ///////////// START LOGS LIST FUNCTIONS /////////////
+    /////////////////////////////////////////////////////
+
+    var defaultOktaDomain;
+
+    function initializeOktaDomain() {

```

Comment:
Noted. 

Do you also prefer const for function definition? Like `const openBackuptaConfigPopup = () => {` instead of `function openBackuptaConfigPopup() {` ?

When you say moving away of jQuery you mean only for fetching? Or also for DOM manipulation?

Reply:
> Do you also prefer const for function definition?

usually, no. just for 1-liners, like https://github.com/gabrielsroka/gabrielsroka.github.io/blob/03cc835adaeccf084fa5149d2f4c3730c965a4fe/rockstar/rockstar.js#L572

see the code.

i'm moving away from jQ for everything. starting with fetch.

> delay the initialization

i hadn't thought of that. but don't you need it before you continue? or is it happening "in the background" ?

note, i'm fetching the actual `/default`, so you don't need the `.find(`.

and i forgot about not having top-level `await`. i'm new to reviewing other people's code.